codejail/jail_code.py

```diff
@@ -57,12 +298,6 @@ def is_configured(command):
     """
     return command in COMMANDS
 
-# By default, look where our current Python is, and maybe there's a
-# python-sandbox alongside.  Only do this if running in a virtualenv.
-if hasattr(sys, 'real_prefix'):
-    if os.path.isdir(sys.prefix + "-sandbox"):
-        configure("python", sys.prefix + "-sandbox/bin/python", "sandbox")
-
```

Comment:
This code looked dangerous and anti-patternish to me.    It makes several assumptions that could impact security: 

* The -sandbox virtualenv is created by the owner of the main virtualenv
* The -sandbox virtualenv doesn't contain malicious code
* A user named "sandbox" exists
* The user named "sandbox" doesn't have excessive permissions.
* The user wants a sandbox named "python". (This was previously necessary to run safe_exec, but isn't anymore).

After discussion with @maxrothman, we decided to kill it, on the assumption that calling code should be explicitly configuring its code jails.

@jibsheet: Max said you have tire fire gifs that may be appropriate here.